src/util/getCliOptions.ts

```diff
@@ -18,10 +19,12 @@ const STORYBOOK_RUNNER_COMMANDS: StorybookRunnerCommand[] = [
   'configDir',
   'browsers',
   'eject',
+  'url',
 ];
 
 export const defaultRunnerOptions: CliOptions['runnerOptions'] = {
   configDir: '.storybook',
+  url: 'http://localhost:6006',
```

Comment:
Same q as above--can we specify in `commander` and get CLI docs for free on these default values?